modules/KoreBuild.Tasks/ComputeManyChecksums.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Build.Framework;
+
+namespace KoreBuild.Tasks
+{
+    /// <summary>
+    /// Computes the hash value for many files and updates the metadata of an item with its value.
+    /// </summary>
+    public class ComputeManyChecksum : Microsoft.Build.Utilities.Task
+    {
+        /// <summary>
+        /// A group of files.
+        /// </summary>
+        [Required]
+        [Output]
+        public ITaskItem[] Items { get; set; }
```

Comment:
So this parameter is both the output and the required input? Do we really have a true output here other than editing the files?

Reply:
This is a MSBuild-ism required to update items in-place with new metadata. The expected usage is:

```xml
    <ComputeManyChecksum Items="@(ArtifactInfo)">
      <Output TaskParameter="Items" ItemName="_ArtifactInfoWithFileHash" />
    </ComputeManyChecksum>
```